client/src/controllers/index.ts

```diff
@@ -1,10 +1,12 @@
 import type { Definition } from '@hotwired/stimulus';
 
 import { AutoFieldController } from './AutoFieldController';
+import { ActionController } from './ActionController';
```

Comment:
Would be good to sort these imports alphabetically - will make the file easier to navigate.